### DISCUSSION TITLE
Integration with Atlassian status page?

### DISCUSSION THREAD

**Discussion by woeterman94** [upptime/upptime#930] [Ideas]
Hello,

Would i be possible to do an integration with [Atlassian Status page](https://support.atlassian.com/statuspage/docs/get-started-with-email-automation/)? 
![image](https://github.com/upptime/upptime/assets/5755057/6935b5cd-3fb5-4c71-985b-c73aac129f4b)

This would require me to send a mail to a different e-mail address if a website is not responding. Is this something that is currently possible? From the docs I couldn't find a way to have different notification types for different services / websites.

**Comment by coolusaHD** [comment]
Nice Idea
Would love to work on that.
@AnandChowdhary See https://github.com/upptime/uptime-monitor/pull/255
